src/auth_guards.rs

```diff
@@ -305,7 +306,29 @@ impl<'l> FromRequest<'l> for InvokeAuthWrapper {
                         };
                     let auth_ref = match orbit.capabilities.invoke([token.clone()]).await {
                         Ok(c) => c,
-                        Err(e) => return unauthorized(e),
+                        Err(InvokeError::Unauthorized(e)) => return unauthorized(e),
+                        Err(InvokeError::Other(e)) => {
+                            error!("{}", e);
```

Comment:
Do we want this `error!` or was it just for debugging?

Reply:
It was for debugging but I think we should keep it at least as a warning (9a47939) because we'll uncover a lot of unhandled errors with it.